runtime-modules/content/src/lib.rs

```diff
@@ -1666,7 +1666,17 @@ decl_module! {
         }
 
         /// Issue NFT
-        #[weight = 10_000_000] // TODO: adjust weight
+        ///
+        /// <weight>
+        ///
+        /// ## Weight
+        /// `O (1)`
+        /// - DB:
+        ///    - O(W + B) where:
```

Comment:
Should be `O(W)` (currently it's actually `O(1)`, but this will change once https://github.com/Joystream/joystream/pull/4221 is merged)

Reply:
The weight should be O(W + B) and DB should be O(W) (currently it's the other way around)